{"name": "Авторизация с неверным паролем", "status": "passed", "description": "Проверка ошибки при попытке авторизации с неверным паролем", "start": 1731008777474, "stop": 1731008777527, "uuid": "4476f6fe-1283-4246-8b41-abe51dc6802d", "historyId": "3058b4de507fc2d51e2f0a5f3fee7224", "testCaseId": "3058b4de507fc2d51e2f0a5f3fee7224", "fullName": "tests.couriers.test_login_courier.TestLoginCourier#test_login_with_wrong_password", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-ALVFPCF"}, {"name": "thread", "value": "15732-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_login_courier"}]}